packages/web-react/src/icons/demo/icons.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import { Grid, Text, Icon } from '../../components';
+import IconsContext, { IconsContextType } from '../../context/IconsContext';
+
+const Icons = () => {
+  let icons;
+  const iconsContext: IconsContextType = useContext(IconsContext);
+
+  if (iconsContext !== null && iconsContext !== undefined) {
+    icons = Object.keys(iconsContext);
```

Comment:
Cool. Maybe we will place this somewhere else in the future and use it on argtypes in iconName props.